service/service_setup.go

```diff
@@ -34,21 +34,46 @@ import (
 	"go.uber.org/zap"
 )
 
-func (svc *serviceCore) setupLogging() {
-	err := svc.configProvider.Get("logging").PopulateStruct(&svc.logConfig)
-	if err != nil {
-		zap.L().Info("Logging configuration is not provided, using UberFX defaults", zap.Error(err))
+func (svc *serviceCore) setupLogging() error {
+	cfg := svc.configProvider.Get("logging")
+	if cfg.HasValue() {
```

Comment:
This block can be moved into ulog as `ulog.Configure()` I don't think service setup needs to understand zapCfg as something separate than ulog config.